packages/@atjson/source-html/src/converter/third-party-embeds.ts

```diff
@@ -76,5 +76,35 @@ export default function convertThirdPartyEmbeds(doc: Document) {
       );
     });
 
+  /**
+   *
+   * <fw-embed-feed channel="vanity_fair" playlist="gYNwOv" mode="row"
+   * open_in="_modal" max_videos="0" placement="middle" player_placement="bottom-right"
+   * pip="false" player_minimize="false" branding="false"></fw-embed-feed>
+   *
+   */
+
+  function isFireworkEmbed(a: Annotation<any>) {
+    return a?.type?.toLowerCase().trim() === "fw-embed-feed";
+  }
+
+  doc
+    .where((a) => isFireworkEmbed(a))
+    .update((embed) => {
+      doc.replaceAnnotation(
+        embed,
+        new FireworkEmbed({
+          id: embed.id,
+          start: embed.start,
+          end: embed.end,
+          attributes: {
+            id: embed.id,
```

Comment:
This is pointing to the incorrect id:

```suggestion
            id: embed.attributes.id,
```